lib/rails/observers/railtie.rb

```diff
@@ -20,6 +18,7 @@ class Railtie < ::Rails::Railtie
       end
 
       config.after_initialize do |app|
+        require 'active_record/base'
```

Comment:
Ok. I'll take out the require and perhaps we can debug the dependency error from the resulting backtrace. This fix  had me pretty lost, and I'm normally comfortable with the Rails/Ruby internals.